projects/igniteui-angular/src/lib/chips/chip.component.ts

```diff
@@ -377,7 +378,12 @@ export class IgxChipComponent extends DisplayDensityBase {
      * @internal
      */
     @HostBinding('attr.tabIndex')
-    public hostTabIndex = null;
+    get setTabIndex() {
```

Comment:
The `setTabIndex` and `tabIndex`  getters now have the same logic, which I think can be improved.

Reply:
Moved the HostBinding on the input itself